src/Web/AdminPanel/Styles/Navigation.scss

```diff
@@ -15,7 +15,7 @@ app > nav > div:first-child {
 @media (min-width: $widthThreshold) {
     app > nav {
         width: 250px;
-        height: 100vh;
+        height: 100%;
```

Comment:
This looks good when the page is long, but worse when it's short

Reply:
I'll keep it and add round edges and shadows to compensate that 😅

Reply:
I found a way to get the best of both solutions... it's now always fully visible whether scrolled or not.